src/policy/concrete.rs

```diff
@@ -218,18 +218,18 @@ pub enum PolicyError {
     DuplicatePubKeys,
 }
 
-/// Descriptor context for [`Policy`] compilation into a [`Descriptor`]
+/// Descriptor context for [`Policy`] compilation into a [`crate::Descriptor`].
 pub enum DescriptorCtx<Pk> {
-    /// [Bare][`Descriptor::Bare`]
+    /// See docs for [`crate::Descriptor::Bare`].
```

Comment:
I'm not super happy with all the `crate::`s here.

Reply:
Does anyone remember how to import a type just for the docs build, we do it someplace else but I can't find it?

Reply:
Its
```rust
#[cfg(doc)]
use Foo;
```